tests/foreman/ui/test_errata.py

```diff
@@ -247,6 +254,75 @@ def test_end_to_end(session, module_repos_col, vm):
         assert result['result'] == 'success'
 
 
+@pytest.mark.tier2
+@pytest.mark.skipif((not settings.repos_hosting_url), reason='Missing repos_hosting_url')
+def test_content_host_errata_page_pagination(session, module_org, module_lce):
+    """
+    # Test per-page pagination for BZ#1662254
+    # Test apply by REX using Select All for BZ#1846670
+
+    :Steps:
+        1. Install more than 20 packages that need errata
+        2. View Content Host's Errata page
+        3. Assert total_pages > 1
+        4. Change per-page setting to 100
+        5. Assert table has more than 20 errata
+        6. Use the selection box on the left to select all on the page.
+            The following is displayed at the top of the table:
+            All 20 items on this page are selected. Select all YYY.
+
+        7. Click the "Select all" text and assert more than 20 results are selected.
+        8. Click the drop down arrow to the right of "Apply Selected", and select
+            "via remote execution - customize first "
+        9. Assert all the errata is in the "errata" field.
+        10. Click Submit
+        11. Assert Errata are applied as expected.
+
+    :BZ: 1662254, 184667
```

Comment:
bz#184667 is about AMD processor support in RHEL5. Is that the correct number?

Reply:
1846670, missing 0, copy paste fail